### PR TITLE
Allow to show less ticks than steps and allow to hide pointer labels

### PR DESCRIPTION
**Pull request by fofi**
This allow to add step for ticks in case you want less than steps.
It also add an attribute to hide pointer labels, before always hide pointer labels when tick labels were shown.

### CODE REVIEW

**Comment by ValentinH** [comment]
Could you please set up a JSFiddle showing what feature you have added?

**Comment by fofi** [comment]
Here it is:  http://jsfiddle.net/fofi/7au7vsL6/1/

File: src/rzslider.js [hunk 1]
```diff
@@ -595,14 +602,12 @@ function throttle(func, wait, options) {
         this.hideEl(this.ceilLab);
       }
 
-      if(this.showTicksValue) {
```
Comment (ValentinH) [inline src/rzslider.js]: Why are you removing this part? This will break existing implementations...

**Comment by ValentinH** [comment]
The `rz-slider-hide-pointer-labels` doesn't handle true/false values. If you set false, then the pointer labels are hidden which is not what I would expect.

Btw, I have started some work on the latest version to add the same feature: #264

**Comment by fofi** [comment]
Hi, 
that part of code, auto hide all labels when tick labels were active, so I added the `rz-slider-hide-pointer-labels` to control it.

About the true/false stuff, at least in this version, that occurs to every attribute, and I was not going to fix all that for now, I just add it like it was.
The directive get it 'true' as string and this make `value = !!"false"`, so that ends in true because the string exist.

This probably is due to the directive version we are using, and we will try to upgrade as soon as possible.

**Comment by ValentinH** [comment]
Do you use NPM to grab this library code?

**Comment by fofi** [comment]
Not at the moment, so i think its okey.

**Comment by ValentinH** [comment]
If you don't use it, I would prefer that you use your patched version directly and not update the legacy 1.1.X version. I only keep it in case of a major bug being discovered but I don't want to add new feature to it.

**Pull request closed without merging**